src/Main.cpp

```diff
@@ -139,57 +233,50 @@ int main(int argc, char *argv[])
         delete stream;
     }
 
+
     // Create a window for the game
     s_display = SDL_CreateWindow("imaginaryMyst Alpha", SDL_WINDOWPOS_UNDEFINED, SDL_WINDOWPOS_UNDEFINED, winWidth,
                                  winHeight, SDL_WINDOW_OPENGL);
 
+
     SDL_GLContext glContext = SDL_GL_CreateContext(s_display);
+    glewExperimental = true;
+    if (glewInit() != GLEW_OK) {
+        fprintf(stderr, "Failed to initialize GLEW\n");
+        return -1;
+    }
+    glEnable(GL_DEBUG_OUTPUT);
+    glEnable(GL_DEBUG_OUTPUT_SYNCHRONOUS);
+    glDebugMessageCallback(openglCallbackFunction, nullptr);
```

Comment:
Visual Studio requires `openglCallbackFunction` to be explicitly cast to type `GLDEBUGPROC`.